### PR TITLE
add docs for synchronous POST /route and synchronous POST /vrp

### PR DESCRIPTION
**Pull request by karussell**
first try ...

TODOs:
 
 - [x] make naming between GET, synch POST and async/batch POST more consistent
 - [x] update [intro blog post](https://www.graphhopper.com/blog/2019/05/17/getting-started-with-the-optimization-api-traveling-salesman-problem/) to use new & simpler synchronous POST
 - [x] POST /route with optimize=true does not work yet - document that? https://github.com/graphhopper/directions-api-doc/issues/48

### CODE REVIEW

File: spec/openapi.yaml [hunk 1]
```diff
@@ -1300,44 +1525,96 @@ paths:
...
 
-        ### Getting Started
+        To get started with the Route Optimization API see the [introduction](#tag/Route-Optimization-API).
 
-        This tutorial - [Getting Started with the Optimization API](https://www.graphhopper.com/blog/2019/05/17/getting-started-with-the-optimization-api-traveling-salesman-problem/) - 
-        illustrates the very basic steps and building blocks to model your vehicle routing problem. This only shows you an excerpt of 
-        what is possible with our API, read more about it below. To unfold detailed information about each object, you just need to click on e.g. `services` or `relations` as shown here:
+        The Route Optimization endpoint is used with a POST request towards
+        `https://graphhopper.com/api/1/vrp?key=<your_key>`. Then you'll find the solution in the JSON response.
+        Please note that for problems that take longer than 10 seconds a bad request error is returned.
```
Comment (oblonski) [inline spec/openapi.yaml]: Der Default-Wert ist 30sec. Wollen wir den auf 10sec anpassen oder hier auf 30sec erhöhen?

Comment (karussell) [reply spec/openapi.yaml]: Yes, we should reduce this in our backend to 10 or 12 seconds.

Comment (oblonski) [reply spec/openapi.yaml]: Dann setzte ich auch den Default-Wert mal auf 10sec.

**Comment by karussell** [comment]
> update intro blog post to use new & simpler synchronous POST

Have adapted the blog post and using the non-async variant makes it indeed a bit simpler. Especially for beginners.

**Comment by oblonski** [comment]
👍 

Which blogpost?

**Comment by oblonski** [comment]
Ah the intro post. I see.

**Comment by karussell** [comment]
Sorry, I meant the intro blog post linked in the docs: https://www.graphhopper.com/blog/2019/05/17/getting-started-with-the-optimization-api-traveling-salesman-problem/

But maybe we should create a separate blog post for this new feature here?

**Pull request merged**